account_payment_line_free/models/account_payment_line.py

```diff
@@ -3,12 +3,13 @@
 # @author: David Wulliamoz, Emanuel Cino
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
 
-from odoo import models, fields, exceptions
+from odoo import models, fields, exceptions, api
 
 
 class AccountPaymentLine(models.Model):
     _inherit = 'account.payment.line'
 
+    @api.one
```

Comment:
I would use api.multi as api.one is deprecated and there is a loop over self in the function so it should be perfectly fine.
```suggestion
    @api.multi
```